abi.html

```diff
@@ -4742,9 +4750,12 @@
 </p>
 
 <pre><font color=blue><code>
-  &lt;function-type&gt; ::= [&lt;<a href="#mangle.CV-qualifiers">CV-qualifiers</a>&gt;] [Dx] F [Y] &lt;<a href="#mangle.bare-function-type">bare-function-type</a>&gt; [&lt;<a href="#mangle.ref-qualifier">ref-qualifier</a>&gt;] E
+  &lt;function-type&gt; ::= [&lt;<a href="#mangle.CV-qualifiers">CV-qualifiers</a>&gt;] [&lt;<a href="#mangle.exception-spec">exception-spec</a>&gt;] [Dx] F [Y] &lt;<a href="#mangle.bare-function-type">bare-function-type</a>&gt; [&lt;<a href="#mangle.ref-qualifier">ref-qualifier</a>&gt;] E
   &lt;<a name="mangle.bare-function-type">bare-function-type</a>&gt; ::= &lt;<i>signature</i> <a href="#mangle.type">type</a>&gt;+
 	# types are possible return type, then parameter types
+  &lt;<a name="mangle.exception-spec">exception-spec&gt; ::= Do                # non-throwing exception-specification
+                   ::= DO &lt;<a href="#mangle.expression">expression</a>&gt; E # computed (instantiation-dependent) noexcept
+                   ::= Dw &lt;<a href="#mangle.type">type</a>&gt;+ E      # throw (type(s))
```

Comment:
I would mention "(instantiation-dependent)" here too. `throw()` should be mangled as `Do`, and `throw(anything else)` (as an extension) should be mangled as an empty \<exception-spec\> if it's not instantiation-dependent.